### DISCUSSION TITLE
Add linked PDFs to media archive or further output format

### DISCUSSION THREAD

**Discussion by ga-it** [ArchiveBox/ArchiveBox#951] [Ideas]
Many paywall pages offer a linked PDF of the page's content.

It would be very useful to have these as an output option - either as an addition to the media option or as a new separate format.

Regards

Marc

**Comment by pirate** [comment]
I need specific examples, cant code for "generally any page that links to a PDF" unless you want to use `depth=1`.

**Reply by ga-it** [reply]
Hi Pirate

I am sorry - clearly that was too vague.

Here is a specific example page:

https://www.bcg.com/publications/2020/valuable-productivity-gains-covid-19

This page has a download pdf link:

https://web-assets.bcg.com/2f/4f/acb18f9c4a68b1cf0367479fc44c/bcg-what-12000-employees-have-to-say-about-the-future-aug-2020.pdf

This is common across many media, academic and subscription sites (e.g. HBR, Sloan, etc).

The goal would be to tell Archivebox to archive the page and include any linked PDFs in the media selection (or new category).

**Reply by felipemeres** [reply]
I am looking for the same feature and would really appreciate if this were to become possible!
